package.json

```diff
@@ -54,13 +58,21 @@
     "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-react-hooks": "^2.3.0",
     "jest": "^24.9.0",
-    "prettier": "^1.19.1",
+    "prettier": "@bornfight/prettier",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-redux": "^7.2.0",
     "react-scripts": "^3.4.0",
     "react-test-renderer": "^16.12.0",
     "redux": "^4.0.4",
     "typescript": "~3.8.2"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "format-with-prettier": "prettier --write src/**/*.{ts,tsx}",
+    "cypress-open-integration": "cypress run --spec 'cypress/integration/*'"
   }
```

Comment:
This should be in the `template.json` file. https://github.com/reduxjs/cra-template-redux-typescript/blob/master/template.json

https://create-react-app.dev/docs/custom-templates/#the-templatejson-file